sdk/js/src/EventEmitter.ts

```diff
@@ -20,20 +20,11 @@ const isInvalidEventKey = (key: string): boolean => {
     !key.startsWith(EventNames.NEW_VARIABLES)
 }
 
-const callHandler = (event: Event, handler: eventHandler): void  => {
-    const args = (<CustomEvent<any[]>>event).detail
-    handler(...args)
-}
-
-export class EventEmitter extends EventTarget {
-    events: Record<string, {
-        handler: eventHandler,
-        handlerWrapper: eventHandler
-    }[]>
+export class EventEmitter {
+    handlers: Record<string, eventHandler[]>
```

Comment:
can we update the `eventHandler` type to be `EventHandler`?

Reply:
oh yeah - fixed!